backend/dev.sh

```
@@ -1,2 +1,3 @@
+export CORS_ALLOW_ORIGIN="http://localhost:5173"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)